Helper/Cart.php

```diff
@@ -628,11 +628,13 @@ protected function getCartCacheIdentifier($cart)
         // display_id is always different for every new cart / immutable quote
         // unset it in the cache identifier so the rest of the data can be matched
         unset($cart['display_id']);
+        unset($cart['metadata']['immutable_quote_id']);
```

Comment:
Since we added immutable quote id to cart metadata, our caching was broken on server side.